source/contracts/Controlled.sol

```diff
@@ -36,7 +37,15 @@ contract Controlled is IControlled {
         return true;
     }
 
-    function suicideFunds(address _target) public onlyControllerCaller returns(bool) {
+    function suicideFunds(address _target, IUniverse _universe) public onlyControllerCaller returns(bool) {
```

Comment:
I wonder if we would be better off having this take in an array of ERC20 tokens instead?  That would allow the caller to specify any tokens that the contract somehow got a hold of, including REP from multiple universes, share tokens, Cash, or any other random token.

Reply:
Yeah good call